client/state/terms/selectors.js

```diff
@@ -142,3 +142,21 @@ export function getTerms( state, siteId, taxonomy ) {
 
 	return manager.getItems();
 }
+
+/**
+ * Returns a term for a site taxonomy.
+ *
+ * @param  {Object}  state    Global state tree
+ * @param  {Number}  siteId   Site ID
+ * @param  {String}  taxonomy Taxonomy slug
+ * @param  {Number}  termId   Term ID
+ * @return {?Object}         Term
+ */
+export function getTerm( state, siteId, taxonomy, termId ) {
+	const manager = get( state.terms.queries, [ siteId, taxonomy ] );
+	if ( ! manager ) {
+		return null;
+	}
+
+	return manager.getItem( termId );
```

Comment:
This function will return `undefined` if the term is not known by the manager. Not saying this is a bad thing, but some conventions in selectors around returning `null` for unknown values.